recipe/bld.bat

```diff
@@ -13,33 +13,39 @@ cmake -G "Ninja" ^
       -DARROW_BUILD_STATIC=OFF ^
       -DARROW_BUILD_TESTS=OFF ^
       -DARROW_BUILD_UTILITIES=OFF ^
+      -DARROW_COMPUTE=ON ^
       -DARROW_CSV=ON ^
       -DARROW_CUDA=OFF ^
       -DARROW_DATASET=ON ^
       -DARROW_DEPENDENCY_SOURCE=SYSTEM ^
+      -DARROW_FILESYSTEM=ON ^
       -DARROW_FLIGHT=ON ^
       -DARROW_FLIGHT_REQUIRE_TLSCREDENTIALSOPTIONS=ON ^
       -DARROW_GANDIVA=OFF ^
       -DARROW_HDFS=ON ^
       -DARROW_JSON=ON ^
       -DARROW_MIMALLOC=OFF ^
       -DARROW_OPENSSL_USE_SHARED=ON ^
-      -DARROW_ORC=ON ^
+      -DARROW_PYTHON=ON ^
+      -DARROW_ORC=OFF ^
```

Comment:
Still not clear why we've stopped support ORC on Windows. If we're now getting build failures when building with this feature on, the better course of action may be to fix our ORC builds.

Reply:
Also, the `-DARROW_PYTHON=ON` line should be put back into its original location; that makes the diff much reader to read.